### DISCUSSION TITLE
Enhancing Outline with AI-Powered Collaboration

### DISCUSSION THREAD

**Discussion by jzmini** [outline/outline#6908] [Ideas]
I recently tried Affine's AI chatting feature and was impressed by how seamlessly it integrated advanced language models for natural and intuitive conversations.

Would it be possible to bring a similar AI-powered chatting experience to Outline by integrating with APIs like OpenAI? 

In the long term, it could be beneficial if the AI could summarize our entire knowledge base, searching and summarizing relevant information as needed. This may require significant development efforts, but the potential benefits likely outweigh the challenges involved. As a first step, embedding AI chatting into Outline seems more realistic and achievable.

**Comment by tommoor** [comment]
This feature is already available in the cloud and licensed editions of Outline :) 

https://www.getoutline.com/pricing

**Reply by jzmini** [reply]
I see, the self-hosted community edition doesn't support this feature...
